tst/benchmarking/benchmark_commons/test_hpo_main_local.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+import unittest
+from unittest.mock import MagicMock, patch
+from benchmarking.commons.default_baselines import RandomSearch
+from benchmarking.commons.hpo_main_common import ConfigDict
+from benchmarking.nursery.benchmark_multiobjective.baselines import (
+    Methods,
+    MOREABench,
+    LSOBOBench,
+)
+from benchmarking.nursery.benchmark_multiobjective.benchmark_definitions import (
+    nas201_mo_benchmark,
+)
+from benchmarking.nursery.benchmark_multiobjective.hpo_main import main
+
+
+class HPOMainLocalTests(unittest.TestCase):
```

Comment:
This seems a good effort. We should take this as a start for testing benchmark examples, and more general for integration tests!